src/network/networkd-fdb.c

```diff
@@ -359,3 +361,43 @@ int config_parse_fdb_vxlan_vni(
 
         return 0;
 }
+
+
+int config_parse_fdb_ntf_flags(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        _cleanup_(fdb_entry_free_or_set_invalidp) FdbEntry *fdb_entry = NULL;
+        Network *network = userdata;
+        NeighborCacheEntryFlags f;
+        int r;
+
+        assert(filename);
+        assert(section);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        r = fdb_entry_new_static(network, filename, section_line, &fdb_entry);
+        if (r < 0)
+                return log_oom();
+
+        f = fdb_ntf_flags_from_string(rvalue);
+        if (f < 0)
+                return log_syntax(unit, LOG_ERR, filename, line, f,
+                                  "FDB failed to parse AssociatedWith=, ignoring assignment: %s",
+                                  rvalue);
```

Comment:
Please do not return negative value here. Also, `f` is not negative errno. Then,
```
if (f < 0) {
    log_syntax(..., 0, ...);
    return 0;
}
```